channels/applicationinsights-channel-js/Tests/Sample.tests.ts

```diff
@@ -31,6 +31,7 @@ export class SampleTests extends TestClass {
 
                 Assert.ok(this.sample.isSampledIn(this.item));
                 Assert.ok(scoreStub.notCalled);
+                Assert.ok(!this.item[SampleRate], "sampleRate field is hidden when the sampling percentage is 100");
```

Comment:
Does this test  fail on the other test cases with a <100 sampling rate?

Reply:
Thought about it, it actually doesn't make sense to test it here. SampleRate field will be added in the envelope in "processTelemetry" method of Sender, then serialized off if it equals to 100 in the same function. so removed from here.